mobilizon_reshare/main/publish.py

```diff
@@ -0,0 +1,47 @@
+import logging.config
+from typing import Optional
+
+from mobilizon_reshare.event.event import MobilizonEvent
+from mobilizon_reshare.event.event_selection_strategies import select_event_to_publish
+from mobilizon_reshare.publishers.coordinator import (
+    PublicationFailureNotifiersCoordinator,
+)
+from mobilizon_reshare.publishers.coordinator import PublisherCoordinator
+from mobilizon_reshare.storage.query.read import (
+    get_published_events,
+    build_publications,
+    events_without_publications,
+)
+from mobilizon_reshare.storage.query.write import save_publication_report
+
+logger = logging.getLogger(__name__)
+
+
+async def publish(events: Optional[list[MobilizonEvent]] = None):
+    """
+    STUB
+    :return:
+    """
+    if events is None:
+        events = await events_without_publications()
+
+    event = select_event_to_publish(
+        list(await get_published_events()),
+        events,
+    )
+
+    if event:
+        logger.info(f"Event to publish found: {event.name}")
```

Comment:
What's the purpose? How is it different from retry?

Reply:
This is quite a good point, I hadn't thought about it this way. If you agree I'd discuss how to proceed about this duplication in #158.

Reply:
ok